usagov/vote-gov-drupal
# Feature/vote 2729 nvrf privacy path

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2729

## Description

Adds `data-privacyPath` to the NvrfPageController and adds a rule between the header and the app on the app page

## Deployment and testing

### Post-deploy steps

1. cd into votegov theme and run `npm run build`
2. run `lando retune` 

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/register/alabama/form and confirm there is a line between the header and app
3. click on the privacy policy link and make sure it is taking you to `/form-filler-privacy-policy` (this should be a 404 page because it does not exist. You can create a new basic page at that path and check it again if you would like)

## Checklist for the Developer




## Checklist for the Peer Reviewers
